src/sql/sqlops.proposed.d.ts

```diff
@@ -484,7 +484,14 @@ declare module 'sqlops' {
 		/**
 		 * The languge mode for this text editor. The language mode is SQL by default.
 		 */
-		languageMode?: string
+		languageMode?: string;
+		/**
+		 * The position CSS property for the editor. Empty by default.
+		 * If the editor is included inside a FlexBox container this must be
+		 * set to 'absolute', with the parent FlexBox having 'relative' position.
```

Comment:
When you say "FlexBox" here do you mean our `FlexContainer` or something else? It might help to clarify. In any case is there some reason we can't automatically detect this case and set the position instead of requiring the extension author to do it?

Reply:
I'll change naming to FlexContainer. It's technically any component that has `flex` display. 

Regarding auto-detection: I am not aware of a way for the child container to know that its parent is flex and handle automatically. As far as I know it's not easy.